### PR TITLE
[zh] Kubernetes 1.29: VolumeAttributesClass for Volume Modification

### PR DESCRIPTION
**Pull request by carlory**
Add zh text to this blog:

```
content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md
```

See [preview](https://deploy-preview-44373--kubernetes-io-main-staging.netlify.app/zh-cn/blog/2023/12/15/kubernetes-1-29-volume-attributes-class/) please.

### CODE REVIEW

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+<!--
+## How to use it
+
+If you want to test the feature whilst it's alpha, you need to enable the relevant feature gate
+in the `kube-controller-manager` and the `kube-apiserver`. Use the `--feature-gates` command line argument:
+-->
+## 如何使用它
+
+如果您想在 alpha 版本中测试该功能，需要在 `kube-controller-manager` 和 `kube-apiserver` 中启用相关的特征门控。
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:89]:
```suggestion
如果您想在 Alpha 版本中测试该功能，需要在 `kube-controller-manager` 和 `kube-apiserver` 中启用相关的特征门控。
```
在中文本地化中，“alpha，beta 等版本” 一般首字母大写

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+它还需要 CSI 驱动程序实现 ModifyVolume API。
+
+<!-- 
+### User flow
+
+If you would like to see the feature in action and verify it works fine in your cluster, here's what you can try:
+-->
+### 用户流程
+
+如果您想看到该功能的运行情况，并验证它在您的集群中是否正常工作，可以尝试以下操作：
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:108]:
```suggestion
如果你想看到该功能的运行情况，并验证它在你的集群中是否正常工作，可以尝试以下操作：
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+3. 验证 PersistentVolumeClaim 是否已正确制备：
+
+   ```
+   kubectl get pvc
+   ```
+
+<!--
+4. Create a new VolumeAttributesClass gold:
+-->
+4. 创建一个新的 VolumeAttributesClass gold：
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:170]:
```suggestion
4. 创建一个新的名为 gold 的 VolumeAttributesClass：
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+* See the [VolumeAttributesClass KEP](https://kep.k8s.io/3751) for more information on the design
+* You can view or comment on the [project board](https://github.com/orgs/kubernetes-csi/projects/72) for VolumeAttributesClass
+* In order to move this feature towards beta, we need feedback from the community,
+  so here's a call to action: add support to the CSI drivers, try out this feature,
+  consider how it can help with problems that your users are having…
+-->
+## 后续步骤
+
+* 有关设计的更多信息，请参阅 [VolumeAttributesClass KEP](https://kep.k8s.io/3751)
+* 您可以在 [VolumeAttributesClass 项目板](https://github.com/orgs/kubernetes-csi/projects/72) 上查看或评论
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:224]:
```suggestion
* 你可以在 [VolumeAttributesClass 项目看板](https://github.com/orgs/kubernetes-csi/projects/72) 上查看或评论
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+We always welcome new contributors. So, if you would like to get involved, you can join our [Kubernetes Storage Special Interest Group](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG).
+-->
+## 参与其中
+
+我们始终欢迎新的贡献者。因此，如果您想参与其中，可以加入我们的 [Kubernetes 存储特别兴趣小组](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG)。
+
+<!--
+If you would like to share feedback, you can do so on our [public Slack channel](https://app.slack.com/client/T09NY5SBT/C09QZFCE5).
+-->
+如果您想分享反馈意见，可以在我们的 [公共 Slack 频道](https://app.slack.com/client/T09NY5SBT/C09QZFCE5) 上留言。
```
Comment (mengjiao-liu) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:240]:
```suggestion
如果你想分享反馈意见，可以在我们的[公共 Slack 频道](https://app.slack.com/client/T09NY5SBT/C09QZFCE5) 上留言。
```

**Comment by carlory** [comment]
@mengjiao-liu updated.

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+
+**VolumeAttributesClass**
+
+<!-- 
+Represents a specification of mutable volume attributes defined by the CSI driver.
+The class can be specified during dynamic provisioning of PersistentVolumeClaims,
+and changed in the PersistentVolumeClaim spec after provisioning. 
+-->
+表示由 CSI 驱动程序定义的可变卷属性的规范。你可以在 PersistentVolumeClaims 动态制备时指定它，并且
+允许在制备完成后进行更改。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:53]:
```suggestion
表示由 CSI 驱动程序定义的可变卷属性的规约。你可以在 PersistentVolumeClaim 动态制备时指定它，
并且允许在制备完成后在 PersistentVolumeClaim 规约中进行更改。
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 7]
```diff
@@ -0,0 +1,256 @@
...
+
+<!--
+## How to use it
+
+If you want to test the feature whilst it's alpha, you need to enable the relevant feature gate
+in the `kube-controller-manager` and the `kube-apiserver`. Use the `--feature-gates` command line argument:
+-->
+## 如何使用它
+
+如果你想在 Alpha 版本中测试该功能，需要在 `kube-controller-manager` 和 `kube-apiserver` 中启用相关的特征门控。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:89]:
```suggestion
如果你想在 Alpha 版本中测试该功能，需要在 `kube-controller-manager` 和 `kube-apiserver` 中启用相关的特性门控。
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 8]
```diff
@@ -0,0 +1,256 @@
...
+* See the [VolumeAttributesClass KEP](https://kep.k8s.io/3751) for more information on the design
+* You can view or comment on the [project board](https://github.com/orgs/kubernetes-csi/projects/72) for VolumeAttributesClass
+* In order to move this feature towards beta, we need feedback from the community,
+  so here's a call to action: add support to the CSI drivers, try out this feature,
+  consider how it can help with problems that your users are having…
+-->
+## 后续步骤
+
+* 有关设计的更多信息，请参阅 [VolumeAttributesClass KEP](https://kep.k8s.io/3751)
+* 你可以在 [VolumeAttributesClass 项目看板](https://github.com/orgs/kubernetes-csi/projects/72) 上查看或评论
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:224]:
```suggestion
* 你可以在[项目看板](https://github.com/orgs/kubernetes-csi/projects/72)上查看或评论 VolumeAttributesClass
```

File: content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md [hunk 9]
```diff
@@ -0,0 +1,256 @@
...
+  尝试此功能，考虑它如何帮助解决你的用户遇到的问题...
+
+<!--
+## Getting involved
+
+We always welcome new contributors. So, if you would like to get involved, you can join our [Kubernetes Storage Special Interest Group](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG).
+-->
+## 参与其中
+
+我们始终欢迎新的贡献者。因此，如果你想参与其中，可以加入我们的 [Kubernetes 存储特别兴趣小组](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG)。
```
Comment (windsonsea) [inline content/zh-cn/blog/_posts/2023-12-15-volume-attributes-class/index.md:235]:
```suggestion
我们始终欢迎新的贡献者。因此，如果你想参与其中，可以加入我们的
[Kubernetes 存储特别兴趣小组](https://github.com/kubernetes/community/tree/master/sig-storage) (SIG)。
```

**Comment by mengjiao-liu** [comment]
Please squash these commits to one.

**Comment by mengjiao-liu** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**